### PR TITLE
Virtual Device Related Changes

### PR DESCRIPTION
**Pull request by parvezah**

### CODE REVIEW

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/VirtualDevice/NewAzureStorSimpleVirtualDeviceCommand.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+using Microsoft.WindowsAzure.Management.StorSimple.Models;
+
+namespace Microsoft.WindowsAzure.Commands.StorSimple.Cmdlets.VirtualDevice
+{
+    [Cmdlet(VerbsCommon.New, "AzureStorSimpleVirtualDevice")]
+    public class NewAzureStorSimpleVirtualDeviceCommand : StorSimpleCmdletBase
+    {
+        [Parameter(Position = 0, Mandatory = true, HelpMessage = StorSimpleCmdletHelpMessage.VirtualDeviceName)]
+        [ValidateNotNullOrEmpty]
+        public string VirtualDeviceName { get; set; }
```
Comment (avirupch) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/VirtualDevice/NewAzureStorSimpleVirtualDeviceCommand.cs]: Please work with Harsha to have a smaller alias, e.g. Name.

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/VirtualDevice/NewAzureStorSimpleVirtualDeviceCommand.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    ReturnWorkflowId = true,
+                    VirtualNetworkName = VirtualNetworkName,
+                    SubNetName = SubNetName,
+                    CreateNewStorageAccount = CreateNewStorageAccount.IsPresent,
+                    StorageAccountName = StorageAccountName
+                };
+
+                var deviceJobResponse = StorSimpleClient.CreateVirtualDevice(applianceProvisiongInfo);
+
+                HandleDeviceJobResponse(deviceJobResponse, "add");
```
Comment (avirupch) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/VirtualDevice/NewAzureStorSimpleVirtualDeviceCommand.cs]: In general the operation name for a *New* cmdlet is specified as "create"

File: src/ServiceManagement/StorSimple/Commands.StorSimple/StorSimpleCmdletBase.cs [hunk 3]
```diff
@@ -92,6 +92,16 @@ internal virtual void HandleAsyncTaskResponse(AzureOperationResponse opResponse,
             WriteVerbose(msg);
         }
 
+        internal virtual void HandleDeviceJobResponse(JobResponse deviceJobResponse, string operationName)
```
Comment (avirupch) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/StorSimpleCmdletBase.cs]: Consider reusing HandleDeviceJobResponse in https://github.com/AzCiS/azure-powershell/pull/4/files, which also takes care of the failure scenario.

Comment (parvezah) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/StorSimpleCmdletBase.cs]: IMHO, failure scenarios should be handled in the sdk client itself.(and it is handled in the sdk client). Hence we shouldnt worry about it here.

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/VirtualDevice/NewAzureStorSimpleVirtualDeviceCommand.cs (hunk 2 above)
Comment (parvezah) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/VirtualDevice/NewAzureStorSimpleVirtualDeviceCommand.cs]: i looked at new backup policy and picked the name. Will change to create.

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/VirtualDevice/NewAzureStorSimpleVirtualDeviceCommand.cs (hunk 1 above)
Comment (parvezah) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/VirtualDevice/NewAzureStorSimpleVirtualDeviceCommand.cs]: Done

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/VirtualDevice/NewAzureStorSimpleVirtualDeviceCommand.cs (hunk 2 above)
Comment (parvezah) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/VirtualDevice/NewAzureStorSimpleVirtualDeviceCommand.cs]: Changed to Create

**Pull request merged**